### PR TITLE
Bugfix: Angular Velocity Gradient

### PR DESCRIPTION
**Pull request by Awesomeplayer165**

### CODE REVIEW

File: src/components/config/robotconfig/PathGradient.tsx [hunk 1]
```diff
@@ -87,10 +87,8 @@ class PathGradientFunctions {
     arr: SavedTrajectorySample[],
     documentModel: IStateStore
   ) {
-    const colorValue = Math.abs(
-      Math.sin(point.angularVelocity / 0.00000000001) * 40
-    );
-    return `hsl(${colorValue}, 100%, 50%)`;
+    const normalizedAngularVelocity = (Math.abs(point.angularVelocity) * 100) / 2;
```
Comment (calcmogul) [inline src/components/config/robotconfig/PathGradient.tsx:90]: What are the `* 100` and `/ 2` for?

Comment (Awesomeplayer165) [reply src/components/config/robotconfig/PathGradient.tsx]: 100 is about green ish. 2 is the x max value for angular velocity. higher than that and it shouldn't go more than green.

**Review by calcmogul — COMMENTED** [review]
What do each of the magic numbers do?

**Comment by Awesomeplayer165** [comment]
> What do each of the magic numbers do?

Resolved in discord. I can work on adding documentation to all of pathgradient in a separate PR.

**Review by calcmogul — APPROVED** [review]

**Pull request merged**